recipes/default.rb

```diff
@@ -187,3 +187,12 @@
         node.set['phabricator']['storage_upgrade_done'] = true
     end
 end
+
+# Setup ssh repo hosting if we want it!
+# Only applicable on 14.04 or newer, due to need for AuthorizedKeysCommand which came in
+# OpenSSH 6.2.
+if node['platform_version'] >= '14.04'
```

Comment:
Setting `['phabricator']['vcs_ssh']['hosting_enabled']` to `true` on a system other than (Ubuntu) 14.04 will fail silently. It will be better to fail fast, throwing an exception in the `vcs_ssh_hosting` recipe if the system is unsupported.

Furthermore, what do you think about actually *not* including the `vcs_ssh_hosting` recipe here, but instead including it explicitly in the run list if wanted? Seems easier to grasp what is actually happening.